internal/cloud/awscloud/awscloud.go

```diff
@@ -21,6 +22,7 @@ import (
 type AWS struct {
 	uploader *s3manager.Uploader
 	ec2      *ec2.EC2
+	ec2md    *ec2metadata.EC2Metadata
```

Comment:
Might be easier to read later if `ec2md` was `ec2metadata` since that's what it actually is.